rts/ExternalAI/SSkirmishAICallbackImpl.cpp

```diff
@@ -1912,6 +1912,10 @@ EXPORT(float) skirmishAiCallback_Map_getGravity(int skirmishAIId) {
 	return skirmishAIId_callback[skirmishAIId]->GetGravity();
 }
 
+EXPORT(float) skirmishAiCallback_Map_getWaterDamage(int skirmishAIId) {
+	return mapInfo->water.damage * ((float)GAME_SPEED / UNIT_SLOWUPDATE_RATE);
```

Comment:
imo changing water.damage in SSkirmishAICallbackImpl.cpp is wrong. it should return the "plain" water.damage. shouldn't ai only care if water.damage > 0 or water.damage <= 0?!